pact-jvm-consumer-kotlin/README.adoc

```diff
@@ -0,0 +1,156 @@
+= kPact - Kotlin idiomatic DSL for Pact
+:p: icon:check[role="green"]
+:n: icon:times[role="red"]
+
+== Goals
+* Easy readable
```

Comment:
The prose could be more extensive here, e.g. the resulting pact should be readable. Also should they be easy or simple https://www.youtube.com/watch?v=34_L7t7fD_U

Reply:
done
I reformulated the first two bullet points